packages/muelu/doc/UsersGuide/masterList.xml

```diff
@@ -608,6 +608,14 @@
       <comment-ML>parameter not existing in ML</comment-ML>
     </parameter>
 
+    <parameter>
+      <name>aggregation: greedy Dirichlet</name>
+      <type>bool</type>
+      <default>false</default>
+      <description>Force the node to be Dirichlet if any DOFs on the node are Dirichlet (default is nodes are Dirichlet only if all DOFs on node are Dirichlet).</description>
```

Comment:
Nitpick -- someone might confuse the use of node here with compute node, when what's really meant is aggregate.